habitat/plan.ps1

```diff
@@ -104,12 +104,13 @@ function Invoke-Build {
         }
         Write-BuildLine " ** Running the chef project's 'rake install' to install the path-based gems so they look like any other installed gem."
         $install_attempt = 0
-        do {
-            Start-Sleep -Seconds 5
-            $install_attempt++
-            Write-BuildLine "Install attempt $install_attempt"
-            bundle exec rake install:local --trace=stdout
-        } while ((-not $?) -and ($install_attempt -lt 5))
+        bundle exec rake install --trace=stdout # this needs to be 'bundle exec'd because a Rakefile makes reference to Bundler
```

Comment:
I don't think this file is related to the backport?

Reply:
Yeah, it was for windows plan runner to spit out errors in verify pipeline.

Reply:
Latest update: Windows plan issue is fixed. But I have updated this code block to retry installing gems once, instead of 5 time in a loop. That just increases build time and prevents us from seeing error sooner. If it did not get installed after 2 attempts, it is not going to.